crux-llvm/test/Test.hs

```diff
@@ -79,28 +88,12 @@ data VersionCheck = VC String (Either T.Text Versioning)
 showVC :: VersionCheck -> String
 showVC (VC nm v) = nm <> " " <> (T.unpack $ either id prettyV v)
 
-vcTag :: VersionCheck -> String
-vcTag v@(VC nm _) = nm <> vcMajor v
-
-vcMajor :: VersionCheck -> String
-vcMajor (VC _ v) = either T.unpack (^. major . to show) v
-
 vcVersioning :: VersionCheck -> Either T.Text Versioning
```

Comment:
With the addition of the VersionCheck module, should this just import that module and use its capabilities instead of duplicating them here, or is there a reason that would be problematic that I'm not seeing?

Reply:
Not worth it right now, but something to note for the future.

Reply:
I've left a note about this in https://github.com/GaloisInc/crucible/issues/1044#issuecomment-1535335352.